### PR TITLE
Sdk versioning guide

### PR DESCRIPTION
**Pull request by bennettgo**
### Which issues does this fix?
Release in tandem with SDK versions workato/workato#21680

### PR submission checklist
- [ ] Optimised every new image
- [x] Checked all content for grammatical errors (using Grammarly, for example)
- [X] Added new files to SUMMARY.md

### CODE REVIEW

**Comment by bennettgo** [comment]
looking to get this approved and timed for release with Versioning

File: developing-connectors/sdk.md [hunk 1]
```diff
@@ -6,21 +6,29 @@ date: 2019-05-08 17:00:00 Z
 [![Workato](/assets/images/workato_developer_program.png)](https://www.workato.com)
 
 # Introduction
-Here you will find the documentation you need to build custom connectors to applications Workato currently doesn't support using our SDK. Creating your own connector is normally only necessary when our platform has not yet built a connector for the application you are looking for or we dont support the triggers or actions you need. Building a custom connector is made much easier with basic understanding of coding and allows you to connect to any application that has an API exposed. Workato also has custom actions and our own HTTP connectors that might be more suitable if your integration needs aren't so complex. [Learn more about whether to use the SDK, custom actions or our HTTP connection for your integration needs.](/developing-connectors.md)
+Here you will find the SDK documentation you need to build custom connectors to applications Workato currently doesn't support just yet. Building a great connector on Workato means keeping managing both technical aspects as well as the user experience that you provide to those using your connector.
```
Comment (eeshansim) [inline developing-connectors/sdk.md]: Here you will find the SDK documentation you need to build custom connectors to applications Workato currently doesn't support just yet. Building a great connector on Workato means ~keeping~ managing both technical aspects as well as the user experience that you provide to those using your connector.

File: developing-connectors/sdk/quick-Start.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+In the main Workato recipes page, head over to the tools tab and click on connector SDK. This will take you to your registry of custom connectors - a page where you can see all custom connectors that you have built or have cloned from others. If you're signed in, [click here](https://www.workato.com/custom_adapters) to be taken to that page.
+
+![Navigating to SDK](/assets/images/sdk/Navigating-to-SDK.png)
+*Located in the top navigation bar in Workato. This might look different depending on your plan.*
+
+### Creating a new custom connectors
+In the top right corner of the page, you'll see a button to create a new custom connector. Clicking this will take you to your new custom connector's home page. This page will be where you can begin writing custom connector code and find useful information about it like its share link and version history.
+
+### Changing your connector title, description, and logo
+When first creating your custom connector, it is given a default title, logo and empty description. You should populate these fields first to make it easier to search for your customer connector when looking through your registry in the future.
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: When first creating your custom connector, it is given a default title, logo and empty description. You should populate these fields first to make it easier to search for your ~customer~ custom connector when looking through your registry in the future.

File: developing-connectors/sdk/quick-Start.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+Custom connector title changes can be done by clicking on the pencil icon next to the customer connector and editing the name in the box that appears. Saving your changes can be done by clicking the green tick when you are done. Clear any unsaved changes by clicking the grey cross instead.
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: Custom connector title changes can be done by clicking on the pencil icon next to the ~customer~ custom connector and editing the name in the box that appears. Saving your changes can be done by clicking the green tick when you are done. Clear any unsaved changes by clicking the grey cross instead.

File: developing-connectors/sdk/quick-Start.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+  <thead>
+    <tr>
+        <th width='10%'>Function</th>
+        <th width='20%'>Hot-key</th>       
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>Persistent Search</td>
+      <td><code>Ctrl-F / Cmd-F</code></td>
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: <td><code>Ctrl-F</code> / <code>Cmd-F</code></td>

File: developing-connectors/sdk/quick-Start.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+    </tr>
+    <tr>
+      <td>Replace</td>
+      <td><code>Shift-Ctrl-F / Cmd-Option-F</code></td>
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: same as above

File: developing-connectors/sdk/quick-Start.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+    </tr>
+    <tr>
+      <td>Replace all</td>
+      <td><code>Shift-Ctrl-R / Shift-Cmd-Option-F</code></td>
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: same as above

File: developing-connectors/sdk/quick-Start.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+
+Find out more about configuring a connection in our SDK and what connection types we support. [Learn more](/developing-connectors/sdk/authentication.md)
+
+### Testing actions and triggers
+After establishing a successful connection, you'll be able to test any actions and triggers you have defined using the debugger console.
+
+![Testing actions or triggers](/assets/images/sdk/testing-actions-triggers.png)
+*Open up the dropdowns for triggers or actions to see the full list available in your latest version*
+
+When attempting to test and action or trigger, you'll need to define the input parameters expected. This is done via a popup when pressing the "Test" button next to any action or trigger. When testing actions, we immediately populate the JSON with a skeleton to help you along. When you run the action or trigger, it is executed based on the latest version in the code editor and the output is shown in the console to the right of the debugger.
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: When attempting to test ~and~ an action or trigger, you'll

File: developing-connectors/sdk/quick-Start.md [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+
+>Be sure to explore any dependencies that your existing recipes in your production account might have when importing a manifest that overwrites a connector.
+
+## Deleting a custom connector
+A Workato account can have any number of custom connectors in its registry. However, in the case that you do need to delete a custom connector, you'll be prevented from doing so if any active running recipes are using it. You'll need to stop the recipes listed to delete this connector.
+
+![Delete prevention popup](/assets/images/sdk/prevent-delete-popup.png)
+*Stop or edit these recipes to delete a custom connector*
+
+You'll be reminded again of any stopped recipes that will be affected by the deletion of this custom connector before you delete it. Any of this custom connector's cloned children will not be deleted.
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: ...Any ~of this~ custom connector(s) ~'s~ cloned from this ~children~ will not be deleted.

File: developing-connectors/sdk/quick-Start.md [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+
+### Exporting a manifest
+When exporting a manifest, the **latest released version** of your custom connector that is used in recipes is exported as well. [Find out more about exporting here](/recipe-development-lifecycle/export.md)
+
+### Importing a manifest
+When importing a manifest, Workato first checks to see if you have an existing copy of the custom connector. If no existing copy is found, a new one is created with the latest released version as version 1. If an existing copy is found, a new latest version is created instead. In both cases, importing a manifest will immediately release the latest version of the custom connector for you since the recipes you are importing will be using that latest version. [Find out more about importing here](/recipe-development-lifecycle/import.md)
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: finish the last sentence with a period (`.`) even if it's a hyperlink.

File: developing-connectors/sdk/quick-Start.md [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+
+When a child custom connector is updated, a new latest version is created. This version is identical to the latest released version of the parent connector. At this stage, you'll be able to view the source code, the new actions or triggers and decide whether you'd like to release this new version to your recipes. In cases where you felt this update was unnecessary, you'd be able to revert to a previous version in the same way you would restore an old version.
+
+## Exporting packages with custom connectors
+[Recipe lifecycle management](/recipe-development-lifecycle.md) is a tool used to export and import entire folders of recipes from a sandbox environment to a production environment in Workato. Often, recipes that you want to export from one account and import into another contain custom connectors that you have built or cloned. When exporting these recipes, Workato helps to export any custom connectors in the manifests as well.
+
+### Exporting a manifest
+When exporting a manifest, the **latest released version** of your custom connector that is used in recipes is exported as well. [Find out more about exporting here](/recipe-development-lifecycle/export.md)
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: finish the last sentence with a period (`.`) even if it's a hyperlink.

File: developing-connectors/sdk/quick-Start.md [hunk 11]
```diff
@@ -0,0 +1,198 @@
...
+
+##### 7. I don't have teams enabled on my Workato account. How can I test a specific version of my custom connector in recipes before releasing them to all my active recipes?
+We are currently working on improving the coding experience further on the SDK platform to include the ability to run a variety of automated tests on different versions of your custom connector.
+
+In the meantime, a valid workaround would be to create a dummy custom connector which hosts the same code as the version latest stable version. You may introduce improvements to this dummy custom connector and test it separately with dedicated recipes. We would suggest testing this new version on copies of both existing recipes that use the custom connector and new recipes to ensure there are no regressions.
+
+## Building your custom connector
+Now that you're familiar with the platform, its time for you to build your custom connector! Check out our articles on the SDK conceptual model to get you going. [Learn more](/developing-connectors/sdk/SDK-conceptual-model.md)
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: finish the last sentence with a period (`.`) even if it's a hyperlink.

File: developing-connectors/sdk.md [hunk 12]
```diff
@@ -6,21 +6,29 @@ date: 2019-05-08 17:00:00 Z
...
 
 ## What is a custom connector?
 A connector allows Workato to interact with a single application through a series of [triggers](/recipes/triggers.md) and [actions](/recipes/actions.md). Triggers monitor for events that occur in the application you hope to connect to and kickstart a workflow of actions which we call a [recipes](/workato-concepts.md#recipes). Actions carry out specific pre-defined operations in the target application.
 
 Connectors built on the SDK are called **custom connectors**. These connectors have a private scope by default which means that they are only visible and available to the connector owner. After the connector is built and ready, you also have the ability to share it with others on various levels.
 
+## Before we begin...
+Workato has a whole host of other features and functionalities that might help you acheive what you are hoping for such as our universal HTTP connector or custom actions if your integration needs aren't so complex.[Learn more](/developing-connectors.md)
```
Comment (eeshansim) [inline developing-connectors/sdk.md]: finish the last sentence with a period (.) even if it's a hyperlink.

File: developing-connectors/sdk/quick-Start.md [hunk 13]
```diff
@@ -0,0 +1,198 @@
...
+When using the debugger console, the first thing you need to do is establish a successful connection. After defining the connection inside the source code, you'll be testing this connection immediately in the connection tab of the debugger console.
+
+![Establishing connection](/assets/images/sdk/establishing-connection.gif)
+*Successfully establishing a connection*
+
+When the connection you make is successful, the debugger will immediately display a "Connection success" message to let you know. This connection you just established will be used as the connection for any tests that you execute in the debugger console following this.
+
+>You can't test actions or triggers until a successful connection is made. Without a successful connection, HTTP requests sent during triggers and actions would not be able to actively authenticate itself with the API endpoint.
+
+Find out more about configuring a connection in our SDK and what connection types we support. [Learn more](/developing-connectors/sdk/authentication.md)
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: finish the last sentence with a period (.) even if it's a hyperlink.

File: developing-connectors/sdk/quick-Start.md [hunk 14]
```diff
@@ -0,0 +1,198 @@
...
+Update notifications are always pegged to the latest released version of its parent connector. Choosing to update always create a new version on top of your custom connector. Be sure to verify that the newly updated connector has changes that make sense to you before releasing. You can always make edits to the new connector code to suit your purpose.
+
+##### 6. How do I know what changes occur across each version and who created that version when working in a team workspace?
+Our version history table gives you insight into the actions and triggers present in each version. Our versions table also showcases the user who created a version and who released a certain version. More functionality will come to showcase more fine-grained details about specific changes in the code across versions and the ability to attach notes to different versions. Stay tuned!
+
+##### 7. I don't have teams enabled on my Workato account. How can I test a specific version of my custom connector in recipes before releasing them to all my active recipes?
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: I don't see the reason for the first part of this question. What does teams feature have to do with testing a specific version?

File: developing-connectors/sdk/quick-Start.md [hunk 15]
```diff
@@ -0,0 +1,198 @@
...
+We are currently working on improving the coding experience further on the SDK platform to include the ability to run a variety of automated tests on different versions of your custom connector.
+
+In the meantime, a valid workaround would be to create a dummy custom connector which hosts the same code as the version latest stable version. You may introduce improvements to this dummy custom connector and test it separately with dedicated recipes. We would suggest testing this new version on copies of both existing recipes that use the custom connector and new recipes to ensure there are no regressions.
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: In the meantime, a valid workaround would be to create a dummy custom connector ~which~ that hosts the same code as the ~version~ latest stable version...

File: developing-connectors/sdk/quick-Start.md [hunk 16]
```diff
@@ -0,0 +1,198 @@
...
+Workato's SDK platform has numerous features that enable you to build, test, develop and share your custom connector. In this article, we will be going through the basic functionalities of the platform and how you can leverage different functionalities to make your connector development process smoother.  
+
+## Before you begin
+### Navigating to the SDK platform homepage
+In the main Workato recipes page, head over to the tools tab and click on connector SDK. This will take you to your registry of custom connectors - a page where you can see all custom connectors that you have built or have cloned from others. If you're signed in, [click here](https://www.workato.com/custom_adapters) to be taken to that page.
+
+![Navigating to SDK](/assets/images/sdk/Navigating-to-SDK.png)
+*Located in the top navigation bar in Workato. This might look different depending on your plan.*
+
+### Creating a new custom connectors
```
Comment (eeshansim) [inline developing-connectors/sdk/quick-Start.md]: ### Creating a new custom connector ~conenctors~

**Pull request merged**